Assets/VRTK/Prefabs/Resources/Sprites.meta

```diff
@@ -0,0 +1,9 @@
+fileFormatVersion: 2
```

Comment:
Ah the reason why I didn't do that was because it isn't really an example, the locomotion doesn't really work without some sprite being in there, since they cant see where the movement is heading. What about another folder called Images under resources?